new_platforms/vscode-extension/README.md

```diff
@@ -142,6 +142,18 @@ data to Microsoft, you can set the `telemetry.enableTelemetry` setting to `false
 ## Known Issues
 
 * Building SGX enclaves is not currently supported.
+* We've had reports that downloading the SDK from git can slow from within the extension, to workaround any issue here, you can run these commands
```

Comment:
grammar: "workaround" should be "work around" (2 words) when used as a verb

Reply:
ok